proofs/signatures/drat.plf

```diff
@@ -29,10 +29,20 @@
 ; We could sort them, but that would require defining an order on our variables,
 ; and incurring the cost of sorting.
 ; Instead, we do the following:
-;  1. Sweep the first clause, marking variables with flags 1 (pos) and 2 (neg)
-;  2. Sweep the second clause, erasing marks.
+;  1. Sweep the first clause, marking variables with flags 1,3 (pos) and 2,4 (neg)
+;  2. Sweep the second clause, erasing marks 1,2.
 ;  3. Unsweep the first clause, returning FALSE on marks.
```

Comment:
Returning FALSE on marks 3,4? Or also on 1,2?

Reply:
Good question!
Yeah -- FALSE is returned on marks 1,2 but not 3,4. I'll fix the documentations.